src/KKManager/ModpackTool/Data/ModpackToolConfiguration.cs

```diff
@@ -15,32 +15,44 @@ namespace KKManager.ModpackTool
 {
     public class ModpackToolConfiguration : IXmlSerializable, INotifyPropertyChanged
     {
-        private static ModpackToolConfiguration _instance;
+        private static ModpackToolConfiguration s_instance;
         private bool _compressPNGs = true;
         private bool _randomizeCABs = true;
 
-        public static ModpackToolConfiguration Instance => _instance ??= new ModpackToolConfiguration();
+        public static ModpackToolConfiguration Instance
+        {
+            get
+            {
+                if (s_instance == null)
+                {
+                    s_instance = new ModpackToolConfiguration();
+                }
+                return s_instance;
+            }
+        }
```

Comment:
Why change to use explicit code?

Reply:
I tend to refactor things to help me understand it. Since I was modifying the fluentftp I ended up modifying a lot of the file as I went. You can just run your code cleanup util and it'll reset it to your .editorconfig you have defined. Otherwise I'll return it when I get some time.